Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/SubmissionQuota.kt

```diff
@@ -86,6 +86,10 @@ class SubmissionQuota @Inject constructor(
 
     companion object {
         @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+        /**
+         * This quota changes when using ExposureWindow mode from 20 to 6 per day
+         * See: https://developers.google.com/android/exposure-notifications/release-notes
+         */
```

Comment:
Why did we not change this from 20 to 6 in code?

Reply:
_long answer_: when it was implemented, it was unclear whether the old mode / provideDiagnosisKey has to be supported too.
_short answer_: adjusted (PR #1656 )